app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post, class_name: 'Post', foreign_key: 'post_id', counter_cache: true
+
```

Comment:
- [ ] This needs to belong to `:author`, not `:user` according to the ERD.
- [ ] Also, `:post` does not need a specific class name, because the name is the same.
This is a simple way, that meets all requirements:

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```